src/Planners/tests/CMakeLists.txt

```diff
@@ -24,10 +24,29 @@ add_bipedal_test(
 
 if (FRAMEWORK_COMPILE_Unicycle)
 
+  include_directories(${CMAKE_CURRENT_BINARY_DIR})
+  configure_file("${CMAKE_CURRENT_SOURCE_DIR}/FolderPath.h.in" "${CMAKE_CURRENT_BINARY_DIR}/FolderPath.h" @ONLY)
+
+  # Get the urdf model of the robot
+  set(ERGOCUB_MODEL_EXPECTED_MD5 7d24f42cb415e660abc4bbc8a52d355f)
+  if (EXISTS "${CMAKE_CURRENT_BINARY_DIR}/model.urdf")
+    file(MD5 "${CMAKE_CURRENT_BINARY_DIR}/model.urdf" ERGOCUB_MODEL_CHECKSUM_VARIABLE)
+    string(COMPARE EQUAL ${ERGOCUB_MODEL_CHECKSUM_VARIABLE} ${ERGOCUB_MODEL_EXPECTED_MD5} ERGOCUB_MODEL_UPDATED)
+  else()
+    set(ERGOCUB_MODEL_UPDATED FALSE)
+  endif()
+
+  if(NOT ${ERGOCUB_MODEL_UPDATED})
+    message(STATUS "Fetching ergoCubSN000 model from icub-tech-iit/ergocub-software...")
+    file(DOWNLOAD https://raw.githubusercontent.com/icub-tech-iit/ergocub-software/f28733afcbbfcc99cbac13be530a6a072f832746/urdf/ergoCub/robots/ergoCubSN000/model.urdf
+      ${CMAKE_CURRENT_BINARY_DIR}/model.urdf
+      EXPECTED_MD5 ${ERGOCUB_MODEL_EXPECTED_MD5})
+  endif()
```

Comment:
Could you explain  me why we need the model of the robot?

Reply:
Not blocking the PR, but we could also just consider to have a runtime test-only dependency on ergocub-software to have the model, and if necessary get the robot via https://github.com/ami-iit/resolve-robotics-uri-cpp .

Reply:
The CI is failing in the build stage, I cannot reproduce it on my local machine

Reply:
@traversaro fixed it with https://github.com/ami-iit/resolve-robotics-uri-cpp/pull/5.

I have updated the **tag** of `resolve-robotics-uri-cpp` with the commit https://github.com/ami-iit/bipedal-locomotion-framework/pull/844/commits/c2756f1c47803d937d4bb95b40cf25a7891c4e0b

Now I have to check why the unitTest of the `unicycleTrajectoryPlanner` is failing

Reply:
I have switch back to not using `resolve-robotics-uri-cpp` for now, with commit https://github.com/ami-iit/bipedal-locomotion-framework/pull/844/commits/c9a86496e269a1722a132afd1bcceac3426b122d